db/schema.rb

```diff
@@ -1061,6 +1061,19 @@
     t.index ["api_name", "consumer_id", "api_guid"], name: "index_webhooks_subscription", unique: true
   end
 
+  create_table "virtual_agent_user_access_records", force: :cascade do |t|
```

Comment:
You'll also need to check in the db/migrate file you used to make this modification.